packages/cashier/src/Stores/deposit-store.js

```diff
@@ -69,6 +69,11 @@ export default class DepositStore {
                 setTimeoutCashierUrl();
             }
         }
+
+        if (!this.root_store.client.account_status?.status?.includes('deposit_attempt')) {
+            await this.root_store.client.updateAccountStatus();
+        }
+
```

Comment:
this is because when the client visited cashier page at least 1 time, he can no longer change the currency

Reply:
is this condition just to hide the `Manage account` button? i think there is no need for this since `can_change_fiat_currency` is already checked when hiding or showing the button

Reply:
@farrah-deriv yes, you are absolutely right that `can_change_fiat_currency` is already checked when hiding or showing the button.
Also when you visit cashier page for the first time you get `deposit_attempt` account status (and since that! moment Manage account button should not showing), then let's imagine that client move to Reports tab, then he changes his account settings, then he again opens Account switcher and Manage account button still visible, though we know that the client already has `deposit_attempt` account status, but due to the fact that the account statuses are updated only when the page is reloaded, the "Manage account" button will still be available, because `can_change_fiat_currency` hasn't been updated yet because the account statuses haven't been updated.
hope that's clear